api/db/migrations/20241028125952_add-indexes.js

```diff
@@ -0,0 +1,28 @@
+const indexesToAdd = [
+  { tableName: 'certification-subscriptions', index: ['complementaryCertificationId', 'certificationCandidateId'] },
```

Comment:
Il y a deja eu un index de mis sur : api/db/migrations/20240618143134_add-index-on-certification-candidate-id-in-subscriptions-table.js

Est-ce qu'il ne faudrait pas enlever le precedent index (il n'avait pas l'air super efficace) ?

Reply:
Il est utile mais ne couvre pas tous les usages.